README.md

```diff
@@ -168,10 +173,10 @@ class MainActivity: AppCompatActivity() {
 
 ## Online status
 
-Connection status to Chat is available via `ChatDomain.instance().online` which returns a LiveData object you can attach observers to.
+Connection status to Chat is available via `ChatDomain.instance().isOnline()` which returns a Boolean. 
```

Comment:
Why do you want to recommend the one-time check instead of the observable?

Reply:
I thought this has changed. I'll move it back